src/Microsoft.Health.Fhir.Core/Features/Conformance/Serialization/ProfileReferenceConverter.cs

```diff
@@ -0,0 +1,63 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Was this file part of the GraphQL work?

Reply:
**This file is not part of the GraphQL work**, while doing a merge with the main branch, github took the new files from others as my changes.